subo/release/check.go

```diff
@@ -145,7 +145,7 @@ func CheckForLatestVersion() (string, error) {
 	} else if cmdVersion, err := version.NewVersion(SuboDotVersion); err != nil {
 		return "", errors.Wrap(err, "failed to parse current subo version")
 	} else if cmdVersion.LessThan(latestCmdVersion) {
-		return fmt.Sprintf("An upgrade for subo is available: %s → %s\n", cmdVersion, latestCmdVersion), nil
+		return fmt.Sprintf("An upgrade for subo is available: %s → %s\n%s", cmdVersion, latestCmdVersion, InstallInfo(latestCmdVersion)), nil
```

Comment:
Based on my other comment
```suggestion
		return fmt.Sprintf("An upgrade for subo is available: %s → %s\n%s", cmdVersion, latestCmdVersion, InstallInfo()), nil
```